fastapi/routing.py

```diff
@@ -447,13 +473,27 @@ def get_route_handler(self) -> Callable[[Request], Coroutine[Any, Any, Response]
             dependency_overrides_provider=self.dependency_overrides_provider,
         )
 
+        # Awaiting https://github.com/encode/starlette/pull/1444/files to remove this workaround
```

Comment:
I think that is merged and in starlette >= 0.27 which FastAPI master depends on, so this could be removed?